node-hue-api/index.d.ts

```diff
@@ -0,0 +1,1075 @@
+// Type definitions for node-hue-api 2.3.0
+// Project: https://github.com/peter-murray/node-hue-api
+// Definitions by: wind-rider <https://github.com/wind-rider/>, Frederic Morel <https://github.com/fjmorel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+// Functions that are deprecated, not working, or returning callbacks instead of promises were not documented.
+
+declare module "node-hue-api" {
```

Comment:
could you remove this block?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
We recommend to use `ProperModule` in TypeScript 2.0.